### PR TITLE
Update Quickstart for CC Free (beta)

### LINKED ISSUE #9254: Update CC Quickstart for Free Tier
Update [Quickstart](https://www.cockroachlabs.com/docs/cockroachcloud/quickstart.html) steps to reflect new free tier offering

### PR DESCRIPTION
**Pull request by lnhsingh**
Closes #9254, #8028.

### CODE REVIEW

**Comment by lnhsingh** [comment]
@kannanlakshmi - adding you as the reviewer here but let me know if a different PM and/or eng should review. Not sure who should be reviewing for technical accuracy here.

**Review by kannanlakshmi — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9329)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @lnhsingh)

___
*[cockroachcloud/quickstart.md, line 22 at r1](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQr0IQc2TrXa3iVVqGT:-MQr0IQc2TrXa3iVVqGU:bz7y4ra) ([raw file](https://github.com/cockroachdb/docs/blob/4a403874e7f9ec69d2fbeebbf696c2fc102afe6b/cockroachcloud/quickstart.md#L22)):*
> ```Markdown
> 1. [Log in](https://cockroachlabs.cloud/) to your CockroachCloud account.
> 1. On the **Clusters** page, click **Create Cluster**.
> 1. On the **Create your cluster** page, select the **Free Plan**.
> ```

Will we have images here?
___
*[cockroachcloud/quickstart.md, line 36 at r1](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQr2YYF7Lio1_1nw45L:-MQr2YYF7Lio1_1nw45M:b-q4lasz) ([raw file](https://github.com/cockroachdb/docs/blob/4a403874e7f9ec69d2fbeebbf696c2fc102afe6b/cockroachcloud/quickstart.md#L36)):*
> ```Markdown
> Once your cluster is created, the **Connection info** modal displays. Use the information provided in the modal to set up your cluster connection for the SQL user that was created by default:
> 
> 1. Create a `certs` directory on your local machine.
> ```

Can we make this even more foolproof by giving the instruction to create this? I know its outside of CC but given we want to make this as simple/easy as possible, its worth actually showing how to do this. Alternatively or maybe in addition, we could say that when you download your ca.crt it is in your "downloads" folder, you can copy this file to your newly created certs folder. 
___
*[cockroachcloud/quickstart.md, line 43 at r1](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQr3wQc28N7TDiJeg17:-MQr3wQc28N7TDiJeg18:b-ni0g2j) ([raw file](https://github.com/cockroachdb/docs/blob/4a403874e7f9ec69d2fbeebbf696c2fc102afe6b/cockroachcloud/quickstart.md#L43)):*
> ```Markdown
> 
>     {{site.data.alerts.callout_danger}}
>     This connection string contains your password, which will be provided only once. If you forget your password, you will need to reset it.
> ```

"..you can reset it by going to the SQL users tab..."? 
___
*[cockroachcloud/quickstart.md, line 109 at r1](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQr3h2I4KHZtoau3N0f:-MQr3h2I4KHZtoau3N0g:b6emb7g) ([raw file](https://github.com/cockroachdb/docs/blob/4a403874e7f9ec69d2fbeebbf696c2fc102afe6b/cockroachcloud/quickstart.md#L109)):*
> ```Markdown
>     ~~~
> 
>     In the connection string copied from the **Connection info** modal, your username and password are pre-populated. Replace the `<certs_dir>` placeholder with the path to the `certs` directory that you created in [Step 2](#step-2-set-up-your-cluster-connection).
> ```

Will there be images here? If not, can we add step by step images?

**Review by lnhsingh — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9329)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kannanlakshmi)

___
*[cockroachcloud/quickstart.md, line 22 at r1](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQr0IQc2TrXa3iVVqGT:-MQx8rKJ1Q2WC5hdYSTh:b9zyhv7) ([raw file](https://github.com/cockroachdb/docs/blob/4a403874e7f9ec69d2fbeebbf696c2fc102afe6b/cockroachcloud/quickstart.md#L22)):*
<details><summary><i>Previously, kannanlakshmi (Lakshmi Kannan) wrote…</i></summary><blockquote>

Will we have images here?
</blockquote></details>

No, wasn't planning on having images here - images are pretty annoying to keep up-to-date and I prefer to use them sparingly. I'm not sure that it's super helpful for this step.
___
*[cockroachcloud/quickstart.md, line 36 at r1](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQr2YYF7Lio1_1nw45L:-MQxAc3l5FZBPFcip9J_:ba9j6u4) ([raw file](https://github.com/cockroachdb/docs/blob/4a403874e7f9ec69d2fbeebbf696c2fc102afe6b/cockroachcloud/quickstart.md#L36)):*
<details><summary><i>Previously, kannanlakshmi (Lakshmi Kannan) wrote…</i></summary><blockquote>

Can we make this even more foolproof by giving the instruction to create this? I know its outside of CC but given we want to make this as simple/easy as possible, its worth actually showing how to do this. Alternatively or maybe in addition, we could say that when you download your ca.crt it is in your "downloads" folder, you can copy this file to your newly created certs folder. 
</blockquote></details>

Added terminal instructions here and in step 4 for moving the cert to the certs directory
___
*[cockroachcloud/quickstart.md, line 43 at r1](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQr3wQc28N7TDiJeg17:-MQxDiXI9oRFD2J0OxPJ:b8n01hn) ([raw file](https://github.com/cockroachdb/docs/blob/4a403874e7f9ec69d2fbeebbf696c2fc102afe6b/cockroachcloud/quickstart.md#L43)):*
<details><summary><i>Previously, kannanlakshmi (Lakshmi Kannan) wrote…</i></summary><blockquote>

"..you can reset it by going to the SQL users tab..."? 
</blockquote></details>

Updated - I'll update the link later once I add instructions on how to change your password
___
*[cockroachcloud/quickstart.md, line 109 at r1](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQr3h2I4KHZtoau3N0f:-MQx_hGEC7Gy17WbJIpE:b-54qhrw) ([raw file](https://github.com/cockroachdb/docs/blob/4a403874e7f9ec69d2fbeebbf696c2fc102afe6b/cockroachcloud/quickstart.md#L109)):*
<details><summary><i>Previously, kannanlakshmi (Lakshmi Kannan) wrote…</i></summary><blockquote>

Will there be images here? If not, can we add step by step images? 
</blockquote></details>

Wasn't planning on adding images, as mentioned a comment above, but I added an example of what the connection string should look like if you replaced the placeholders. Do you think that is enough? IMO, it's pretty self-explanatory

File: Gemfile.lock [hunk 1]
```diff
@@ -106,6 +106,7 @@ GEM
 PLATFORMS
   ruby
   universal-darwin-19
+  x86_64-darwin-19
```
Comment (rmloveland) [inline Gemfile.lock:109]: Needed this to get something to build?

File: _includes/cockroachcloud/quickstart/set-up-your-cluster-connection.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Click the name of the `cc-ca.crt` to download the CA certificate to your local machine.
+1. Create a `certs` directory on your local machine:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ mkdir certs
+    ~~~
+
+1. Move the downloaded `cc-ca.crt` file to the `certs` directory:
```
Comment (rmloveland) [inline _includes/cockroachcloud/quickstart/set-up-your-cluster-connection.md:11]: Another pattern that works in cases like this and lets users go "full copypasta" with even less thinking is:

```
mkdir foo && cd foo && wget https://lauren.zone/some-cool.file
```

Comment (rmloveland) [reply _includes/cockroachcloud/quickstart/set-up-your-cluster-connection.md:11]: Also less typing for thee

File: cockroachcloud/quickstart.md [hunk 3]
```diff
@@ -112,70 +79,61 @@ For this tutorial, we will use the [`movr` workload](../stable/movr.html) to run
...
     ~~~
 
-4. Use the built-in SQL client to view the database:
+    In the connection string copied from the **Connection info** modal, your username, password and cluster name are pre-populated. Replace the `<certs_dir>` placeholder with the path to the `certs` directory that you created in [Step 2](#step-2-set-up-your-cluster-connection). For example:
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    $ cockroach sql \
-    --url='postgres://<username>:<password>@<global host>:26257/movr?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>'
+    $ cockroach sql --url 'postgres://maxroach:password123@free-tier.gcp-us-central1.cockroachlabs.cloud:26257/test-cluster.defaultdb?sslmode=verify-full&sslrootcert=Users/maxroach/certs/cc-ca.crt'
```
Comment (rmloveland) [inline cockroachcloud/quickstart.md:93]: This path to `Users/maxroach/...` would need a slash at the front as well, no?  E.g. `sslrootcert=/Users/maxroach/...`

File: cockroachcloud/quickstart.md [hunk 4]
```diff
@@ -112,70 +79,61 @@ For this tutorial, we will use the [`movr` workload](../stable/movr.html) to run
...
     ~~~
 
+1. Using built-in SQL client, you can now run [CockroachDB SQL statements](learn-cockroachdb-sql.html):
```
Comment (rmloveland) [inline cockroachcloud/quickstart.md:96]: dropped article "the built-in..."  - also is there a link to the `cockroach sql` doc in cloud world?

Comment (rmloveland) [reply cockroachcloud/quickstart.md:96]: er, can we link to the `cockroach sql` doc *from* cloud world? (I meant)

**Review by rmloveland — APPROVED** [review]
LGTM - I found one thing that might be an error (see the `sslrootcert` comment) but otherwise the rest are just thoughts & feelings for your consideration

**Review by lnhsingh — COMMENTED** [review]
TFTRs!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9329#-:-MR0pHVhEq6033x2xN4R:bz5mair)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kannanlakshmi and @rmloveland)

___
*[Gemfile.lock, line 109 at r5](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQxa6JQMzcNZZQaSpFD-r5-109:-MR0fjYB0YOGfGPZN6i5:b-4ji8wh) ([raw file](https://github.com/cockroachdb/docs/blob/cb48117f952d786bb6ef50975f4f1a089e9a41c2/Gemfile.lock#L109)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Needed this to get something to build?
</blockquote></details>

Yeah, since getting a new machine it's been adding this to my PRs. Removed this change!
___
*[_includes/cockroachcloud/quickstart/set-up-your-cluster-connection.md, line 11 at r5](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQy7SwzoRwpzv9h52_E-r5-11:-MR0n02IA8IdB-nOVyW7:b7x5yyo) ([raw file](https://github.com/cockroachdb/docs/blob/cb48117f952d786bb6ef50975f4f1a089e9a41c2/_includes/cockroachcloud/quickstart/set-up-your-cluster-connection.md#L11)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

Also less typing for thee
</blockquote></details>

Discussed on Slack that this doesn't work since we have the user download the cert from the UI to their local machine
___
*[cockroachcloud/quickstart.md, line 93 at r5](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQoTaASwWoSm14ntxg--r5-93:-MR0ojCV8WLE1HaGRntU:bz4vah0) ([raw file](https://github.com/cockroachdb/docs/blob/cb48117f952d786bb6ef50975f4f1a089e9a41c2/cockroachcloud/quickstart.md#L93)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

This path to `Users/maxroach/...` would need a slash at the front as well, no?  E.g. `sslrootcert=/Users/maxroach/...`
</blockquote></details>

Good catch - added.
___
*[cockroachcloud/quickstart.md, line 96 at r5](https://reviewable.io/reviews/cockroachdb/docs/9329#-MQoTaASwWoSm14ntxg--r5-96:-MR0ndzP74AcDLetgGXL:b-eaq2l8) ([raw file](https://github.com/cockroachdb/docs/blob/cb48117f952d786bb6ef50975f4f1a089e9a41c2/cockroachcloud/quickstart.md#L96)):*
<details><summary><i>Previously, rmloveland (Rich Loveland) wrote…</i></summary><blockquote>

er, can we link to the `cockroach sql` doc *from* cloud world? (I meant)
</blockquote></details>

Yes! Added the link.

**Pull request merged**